jwt/jwt.go

```diff
@@ -20,16 +20,27 @@ package jwt
 import (
 	"gopkg.in/square/go-jose.v2"
 	"gopkg.in/square/go-jose.v2/json"
+	"strings"
 )
 
 // JSONWebToken represents a JSON Web Token (as specified in RFC7519).
 type JSONWebToken struct {
+	err     error
```

Comment:
err was introduced to allow chaining - namely it's used by NestedJSONWebToken.Decrypt, but I can make it return (*JSONWebToken, error) instead.